apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -2,6 +2,8 @@ const EVENTS = {
   // Sign-up flow
   ACCOUNT_TYPE_PICKED_EVENT: 'Account Type Picked',
   SIGN_UP_FINISHED_EVENT: 'Sign Up Finished',
+  SECTION_SETUP_SIGN_IN_EVENT: 'Section Setup Sign In',
+  ABANDON_SECTION_SETUP_SIGN_IN_EVENT: 'Abandon Section Setup Sign In',
```

Comment:
Would it be possible to align between 'abandon' and 'dismiss' throughout this implementation (I think it would be ideal if the event names/testing/function names matched)? I think either is good:)